core/templates/services/contextual/window-dimensions.service.ts

```diff
@@ -38,6 +38,13 @@ export class WindowDimensionsService {
       this.windowRef.nativeWindow.document.body.clientWidth);
   }
 
+  getHeight(): number {
+    return (
+      this.windowRef?.nativeWindow.innerHeight ||
```

Comment:
Can you mention a possible case where `this.windowRef` will be undefined? Is the `?` necessary?
I don't understand why are trying to get height from 3 different parameters? Can you explain it?

Reply:
I can't think of a situation where the windowRef could really be undefined, I think the `?` may not be necessary. As for fetching height from 3 parameters, I used the existing `getWidth` function as a reference: 
![image](https://user-images.githubusercontent.com/73544247/187666969-c1b4fcb4-d79a-49bd-ad2d-8b4412fb0089.png)